addon/components/fd-ember-jstree.js

```diff
@@ -134,10 +141,29 @@ export default Component.extend({
       if (this.get('reloadDataAtClose')) {
         this.get('treeObject').on('close_node.jstree', this._closeNodeTree.bind(this));
       }
+    },
+
+    /**
+      Handles selectNode jsTree.
 
+      @method actions.handleTreeDidSelectNode
+    */
+    handleTreeDidSelectNode(node) {
       let selectNodeAction = this.get('selectNodeAction');
       if (selectNodeAction && typeof selectNodeAction === 'function') {
-        this.get('treeObject').on('select_node.jstree', this._selectNodeTree.bind(this));
+        selectNodeAction(node, this.get('store'));
+      }
+    },
+
+    /**
+      Handles moveNode jsTree.
+
+      @method actions.handleTreeDidMoveNode
+    */
+    handleTreeDidMoveNode(node) {
+      let moveNodeAction = this.get('moveNodeAction');
+      if (moveNodeAction && typeof moveNodeAction === 'function') {
```

Comment:
Можно просто проверять typeof moveNodeAction === 'function'. undefined и null это условие не пройдут
```suggestion
      if (typeof moveNodeAction === 'function') {
```